lua/wire/server/wirelib.lua

```diff
@@ -1536,6 +1536,11 @@ if not WireLib.PatchedDuplicator then
 	end
 end
 
+function WireLib.SoundExists(path)
+	if istable(sound.GetProperties(path)) then return true end
+	if file.Exists("sound/" .. string.gsub(path, "^%W", ""), "GAME") then return true end
```

Comment:
Wouldn't %W match the / and \ characters? Maybe should be `"[^%w\\/]"`

Reply:
the "^" at the start of the lua pattern ensures it only tries to match the first character

Reply:
considering any file can have up to 2 sound characters at once i should probably account for that